integrations/phone/examples/common-interactions/README.md

```diff
@@ -0,0 +1,61 @@
+## Phone Starter Kit
+
+This example contains Watson Assistant phone integration actions that demonstrate how to:
```

Comment:
cut 'how to'

('how to' already starts each bullet)

Reply:
```suggestion
This example contains Watson Assistant phone integration actions that demonstrate:
```